.github/workflows/reusable-build-coverage.yml

```diff
@@ -0,0 +1,59 @@
+name: Reusable Coverage Build
+on:
+  workflow_call:
+    inputs:
+      ros_distro:
+        description: 'ROS2 distribution name'
+        required: true
+        type: string
+      os_name:
+        description: 'On which OS to run the workflow, e.g. ubuntu-22.04'
+        required: false
+        default: 'ubuntu-latest'
+        type: string
+
+env:
+  # this will be src/{repo-owner}/{repo-name}
+  path: src/${{ github.repository }}
+
+jobs:
+  coverage:
+    name: coverage build ${{ inputs.ros_distro }}
+    runs-on: ${{ inputs.os_name }}
+    steps:
+      - uses: ros-tooling/setup-ros@0.7.1
+        with:
+          required-ros-distributions: ${{ inputs.ros_distro }}
+      - uses: actions/checkout@v4
+        with:
+          path: ${{ env.path }}
+      - id: package_list_action
+        uses: ros-controls/ros2_control_ci/.github/actions/set-package-list@ci_lint
```

Comment:
```suggestion
        uses: ros-controls/ros2_control_ci/.github/actions/set-package-list@master
```

Reply:
Suggesting a change to your temporary setup... That's a workflow I'll try to remember :-)